app/views/checkout/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<h2>Welcome to checkout</h2>
+<% @cart.shopping_cart_items.each do |cart| %>
```

Comment:
Why do you use variable name `cart`?  When accessing  objects in a collection of cart items wouldn't `item` be a better name?